google-cloud-storage/lib/google/cloud/storage.rb

```diff
@@ -334,6 +334,44 @@ module Cloud
     # downloaded.read #=> "Hello world!"
     # ```
     #
+    # ## Creating and downloading gzip-encoded files
+    #
+    # When uploading a gzip-compressed file, you should pass
+    # `content_encoding: "gzip"` if you want the file to be eligible for
+    # [decompressive transcoding](https://cloud.google.com/storage/docs/transcoding)
```

Comment:
Is this true given that you are decompressing the file post-download?

Reply:
The statement is correct: Such a file will be *eligible* for decompressive transcoding. I believe a major use case for such files is inclusion in static web sites, in which case the client is a web browser and the GCS decompressive transcoding will be performed.

Reply:
Understood! Always appreciate the feedback. Thanks @quartzmo!